payment/locale/fr/LC_MESSAGES/django.po

```diff
@@ -74,18 +88,31 @@ msgstr ""
 "Voici un bref rappel des conditions obligatoires de l'examen surveillé "
 "pour avoir la possibilité d'obtenir le certificat :"
 
-#: /edx/app/edxapp/fun-apps/payment/templates/payment/base.html:0
+#: /edx/app/edxapp/fun-apps/payment/templates/payment/base-abstract.html:0
 msgid "Home"
 msgstr "Accueil"
 
+#: /edx/app/edxapp/fun-apps/payment/templates/payment/list_orders.html:0
+msgid "Nothing has yet been ordered."
+msgstr "Rien n'a encore été commandé."
```

Comment:
Raison originale: je préfère rester assez neutre sur la terminologie ici pour éviter des aller-retours inutiles (cours certifié, examen certifiant,parcours certifié...). Je n'ai pas l'écran en face de moi, mais il me semble que le lien qui dirigeait vers cette page était "Historique de vos commandes" (sur la dashboard), donc ça semble pas illogique de mettre le mot commande là.
Contre proposition si cela te semble gênant:
"Vous n'avez pas encore souscrit à d'examen certifiant"/"You have not registered to any certified exam"
![selection_e9e0](https://cloud.githubusercontent.com/assets/4372386/14376653/1d8b115a-fd6b-11e5-9bdd-96a343e47340.png)
Mais la question se pose alors du lien : "Historique de vos commandes" qui doit alors être changé en "Paiements pour vos examens" ou quelque chose comme ça.
On se fait peut-être du mal pour pas grand chose. La solution la plus intuitive pour l'utilisateur étant de ne pas avoir le lien sur la dashboard s'il n'y a pas de commandes. Mais j'ai compris que ce n'était pas si simple que ça.